index.html

```diff
@@ -0,0 +1,188 @@
+            in the choice of the right wine glass and food to match...
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">dinner_dining</i>
+          </div>
+          <div class="programCardTitle">
+            Dinner
+          </div>
+          <div class="programCardDescription">
+            Seasonal delicacies produced locally: - entree' of cured meats and cheeses,
+            - first course (pasta), and - main course of choice.
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">music_note</i>
+          </div>
+          <div class="programCardTitle">
+            Live Music
+          </div>
+          <div class="programCardDescription">
+            Live band entertaining with soft jazz. A perfect match to make your experience
+            even more sensational.
+          </div>
+        </li>
+
+        <li class="programItemCard">
+          <div class="programCardIcon">
+            <i class="material-icons">cake</i>
+          </div>
+          <div class="programCardTitle">
+            Dessert
+          </div>
+          <div class="programCardDescription">
+            To end the evening on a sweet note: Tastes of delicious light
+            pastries along with carefully selected dessert wines.
+          </div>
+        </li>
+      </ul>
+      <button type="button" class="button contact desktop-invisible">
+        Book your spot
+      </button>
+      <a href="#" class="button program upper mobile-invisible">See the whole program</a>
+    </section>
+
+    <section class="main__speakers flex flex-ai-c">
+      <h2>Experts</h2>
+      <p class="separator"></p>
+      <ul class="speakersContent flex">
+      </ul>
+      <button type="button" class="button expand upper flex flex-jc-c flex-ai-c">
+        More
+        <i class="material-icons">expand_more</i>
+      </button>
+    </section>
+  </main>
+
+  <footer class="footer mobile-invisible">
+    <div class="sponsors flex flex-ai-c">
+      <h2>Partner</h2>
+      <p class="separatorMedium"></p>
+      <div class="partnersLogoContainer flex flex-jc-c flex-ai-c">
+        <img src="." class="logoPartner1" alt="Partner 1 logo">
+        <img src="." class="logoPartner2" alt="Partner 2 logo">
+        <img src="." class="logoPartner3" alt="Partner 3 logo">
+        <img src="." class="logoPartner4" alt="Partner 4 logo">
+        <img src="." class="logoPartner5" alt="Partner 5 logo">
+      </div>
+    </div>
+    <div class="copyrightsContainer flex">
+      <div class="logoPlaceholder">
+        <img src="img/laFicaia.png" class="hostLogo" alt="Event Host logo">
+      </div>
+      <div class="copyrights flex flex-jc-c">
+        <p>2021 Wine Tasting Association</p>
+        <p>This is a template web site. The information inside are invented.</p>
+      </div>
```

Comment:
- [ ] The mobile view of the homepage is currently not displaying any footer. Please see screenshot below: 
![Screenshot 2021-09-10 at 22-24-23 WTV - Wine Tasting (out in the) Vineyard](https://user-images.githubusercontent.com/70150423/132924103-8b32c6f9-2271-42ba-80a2-b7e6d85a132b.png)

Please, let's fix it.

Reply:
Hi @george-shammar, 
Please note that also the original design does not include a footer in the mobile version of the homepage.
![Mobile-home](https://user-images.githubusercontent.com/81151384/132925951-354d2713-ad47-4fa1-8410-5f2a17964b36.png)